tests/functional/ui/test_health_overview.py

```
@@ -344,6 +411,24 @@ def finalizer():
 
         request.addfinalizer(finalizer)
 
+    @pytest.fixture
+    def scale_mon_teardown(self, request):
+        """
+        Teardown fixture to scale up mon after mock critical alert test completes.
+        """
+
+        def finalizer():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)